### ISSUE TITLE
`npx oxlint@latest --max-warnings=0` doesn't tell the warning details

### ISSUE THREAD

**Issue by himself65** [oxc-project/oxc#1958]
For eslint, they will

```shell
npm-download-stat git:(readme) npx oxlint@latest --max-warnings=0      

Finished in 4ms on 17 files with 79 rules using 12 threads.
Exceeded maximum number of warnings. Found 2.
```

**Comment by Boshen** [comment]
There is a `--deny-warnings` flag for exiting CI with exit code 1.

I'll take a look at the no printing issue.

**Comment by himself65** [comment]
Thanks

**Comment by b4s36t4** [comment]
@Boshen is this issue because we exit early before printing warn details?

https://github.com/oxc-project/oxc/blob/2b7ca5962f7961adac92c5bc23741f65b419c037/crates/oxc_cli/src/result.rs#L68-L82 - ref

**Comment by himself65** [comment]
Looks so, I'd like improve this when I'm avaliable

**Comment by Boshen** [comment]
I think it's this check, I didn't know eslint advices `--max-warnings=0`, which is a weird indirection for aborting CI with 1.

https://github.com/oxc-project/oxc/blob/78916703c50a04c66c149bcb61e2851e9b1d0c81/crates/oxc_diagnostics/src/service.rs#L118
